loops-and-arrays.js

```diff
@@ -42,3 +49,7 @@ const squares = [30, 2, 8, 24, 11];
 // console.log(numbers) geeft: [30, 2, 8, 24, 11];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [27000, 8, 512, 13824, 1331];
+for(i=0; i<squares.length; i++) {
+    squares[i] = squares[i] * squares[i] * squares[i];
```

Comment:
In plaats van `squares[i] * squares[i] * squares[i]` had je de Exponentiation Operator kunnen gebruiken!

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Exponentiation

Reply:
Goeie tip! Wist ik niet. Zie wel in de support tabel dat IE niet ondersteund wordt op desktop, maar dat is na een snelle scan.